docs/articles/documentation/templates/client-scripts/specify-pages.md

```diff
@@ -0,0 +1,11 @@
+You can also specify pages into which a script should be injected. For instance, this enables you to mock browser API on particular pages and use the default behavior everywhere else.
```

Comment:
```suggestion
You can also specify pages into which a script should be injected. This will allow you to mock browser API on specified pages and use the default behavior everywhere else.
```